### PR TITLE
Local memory integration

### PR DESCRIPTION
**Pull request by Sweekrity-7392**
Create a check on login ->customRoutingLogic 
1) first time user -> firebase.instance.currentUser == null; go to signIn using gmail
2) signed up user -> check on questionnaire
    2.1. questionnaire not answered ->int 0 stored in local memory, thus go to questionnaire
    2.2. questionnaire answered -> int 1 stored in local memory, thus go to dashboard

(sorry, older commits dusare branch ke is pull request mein aare hai and i do not know how to remove them)

### CODE REVIEW

File: lib/user_page/instructions.dart [hunk 1]
```diff
@@ -59,98 +61,103 @@ class instructions_reminder extends StatelessWidget {
...
-                    //       ),
-                    //     );
-                    //   }
-                    //   return available;
-                    // }
-                    );
-              },
-              child: Text(
+              onPressed: (){oneTimeNotify();},
+              //() async {
```
Comment (skmanish) [inline lib/user_page/instructions.dart:65]: Please remove comments below, or say why you want to keep this.

File: lib/user_page/icons.dart [hunk 2]
```diff
@@ -53,17 +56,25 @@ class log_out extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     return IconButton(
-      constraints: const BoxConstraints(maxHeight: 19, maxWidth: 19),
+      // constraints: const BoxConstraints(maxHeight: 19, maxWidth: 19),
```
Comment (skmanish) [inline lib/user_page/icons.dart:59]: Please remove the comment, or say why you want to keep this.

File: lib/user_page/icons.dart [hunk 3]
```diff
@@ -53,17 +56,25 @@ class log_out extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
```
Comment (skmanish) [inline lib/user_page/icons.dart:57]: Is the logout button used anywhere?

**Review by skmanish — COMMENTED** [review]
Address the comments.

File: lib/user_page/instructions.dart (hunk 1 above)
Comment (Sweekrity-7392) [reply lib/user_page/instructions.dart:65]: okay. I will remove it. I wanted to remove it once I was sure this change was working.

File: lib/user_page/icons.dart (hunk 2 above)
Comment (Sweekrity-7392) [reply lib/user_page/icons.dart:59]: okay, will remove it.

File: lib/user_page/icons.dart (hunk 3 above)
Comment (Sweekrity-7392) [reply lib/user_page/icons.dart:57]: yes, in the dashboard, to logout of the user_page.

**Pull request merged**